src/App.jsx

```diff
@@ -1,9 +1,19 @@
-import { HomePage } from './pages';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
```

Comment:
We need to add the netlify config which will define routes (now if you type route in URL it would throw an netlify error)

Reply:
Done